data_extractor/google_search_history/simulation_gsh.py

```diff
@@ -24,6 +25,8 @@
 
 TRANSITION = ("LINK", "GENERATED", "RELOAD")
 
+ZONE = pytz.utc
+
 
 def __create_website(num: int, perc: float, fake=False):
```

Comment:
I don't understand when you use double underscore before a function and when not. What is the use of the double underscore in this case?